stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/adapter/view/internal/MediaAttachmentsGroupView.kt

```diff
@@ -185,6 +185,9 @@ internal class MediaAttachmentsGroupView : ConstraintLayout {
                 stateCopy.viewThree.setImageShapeByCorners(0f, 0f, 0f, bottomLeftCorner)
                 stateCopy.viewFour.setImageShapeByCorners(0f, 0f, bottomRightCorner, 0f)
             }
+            else -> {
+                // has to be exhaustive or ktLint will fail
```

Comment:
I think this is a consequence of running Kotlin 1.6.10:

https://kotlinlang.org/docs/whatsnew16.html#stable-exhaustive-when-statements-for-enum-sealed-and-boolean-subjects